cds_sorenson/utils.py

```diff
@@ -101,8 +103,9 @@ def get_status(job_id):
 
 def _get_preset_config(preset_id):
     """Return preset config based on the preset_id."""
-    PRESETS = current_app.config['CDS_SORENSON_PRESETS']
-    presets_list = [item for presets in PRESETS.values() for item in presets]
-    for preset in presets_list:
-        if preset_id == preset.get('preset_id'):
-            return preset
+    return [
```

Comment:
Firstly, your proposal is more confusing due to ammiguous naming (i.e. the outer dictionary are not the `presets` and the inner are not the `configs`). Secondly, this is not a style of programming but just usage of list comprehensions, which is widely accepted in Python, due to its readability.